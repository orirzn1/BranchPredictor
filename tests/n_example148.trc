8 5 17 3 local_history local_tables using_share_mid
0xb1663c T 0xe7f2b0f8
0xee7c0 T 0xaf52c
0xee7c0 N 0xaf52c
0xb1663c N 0xe7f2b0f8
0xb1663c N 0xe7f2b0f8
0xb1663c T 0xe7f2b0f8
0xb1663c N 0xe7f2b0f8
0xb1663c N 0xe7f2b0f8
0xee7c0 N 0xaf52c
0xc6ddc N 0xf5c
0xee7c0 T 0xaf52c
0xee7c0 N 0xaf52c
0xc6ddc T 0xf5c
0xee7c0 N 0xaf52c
0xc6ddc N 0xf5c
0xc6ddc T 0xf5c
0xc6ddc T 0xf5c
0xee7c0 T 0xaf52c
0xc6ddc N 0xf5c
0xee7c0 T 0xaf52c
0xb1663c T 0xe7f2b0f8
0xc6ddc N 0xf5c
0xee7c0 N 0xaf52c
0xc6ddc T 0xf5c
0xb1663c T 0xe7f2b0f8
0xee7c0 N 0xaf52c
0xee7c0 N 0xaf52c
0xee7c0 T 0xaf52c
0xc6ddc N 0xf5c
0xb1663c N 0xe7f2b0f8
0xc6ddc N 0xf5c
0xee7c0 N 0xaf52c
0xee7c0 N 0xaf52c
0xb1663c T 0xe7f2b0f8
0xee7c0 T 0xaf52c
0xb1663c N 0xe7f2b0f8
0xc6ddc N 0xf5c
0xb1663c T 0xe7f2b0f8
0xb1663c T 0xe7f2b0f8
0xb1663c N 0xe7f2b0f8
0xc6ddc T 0xf5c
0xee7c0 N 0xaf52c
0xee7c0 T 0xaf52c
0xb1663c N 0xe7f2b0f8
0xb1663c N 0xe7f2b0f8
0xc6ddc N 0xf5c
0xee7c0 N 0xaf52c