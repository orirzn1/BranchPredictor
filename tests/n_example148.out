0xb1663c N 0xb16640
0xee7c0 N 0xee7c4
0xee7c0 T 0xaf52c
0xb1663c T 0xe7f2b0f8
0xb1663c T 0xe7f2b0f8
0xb1663c T 0xe7f2b0f8
0xb1663c T 0xe7f2b0f8
0xb1663c T 0xe7f2b0f8
0xee7c0 T 0xaf52c
0xc6ddc N 0xc6de0
0xee7c0 T 0xaf52c
0xee7c0 T 0xaf52c
0xc6ddc T 0xf5c
0xee7c0 T 0xaf52c
0xc6ddc T 0xf5c
0xc6ddc T 0xf5c
0xc6ddc T 0xf5c
0xee7c0 T 0xaf52c
0xc6ddc T 0xf5c
0xee7c0 T 0xaf52c
0xb1663c N 0xb16640
0xc6ddc N 0xc6de0
0xee7c0 T 0xaf52c
0xc6ddc T 0xf5c
0xb1663c N 0xb16640
0xee7c0 T 0xaf52c
0xee7c0 T 0xaf52c
0xee7c0 T 0xaf52c
0xc6ddc N 0xc6de0
0xb1663c N 0xb16640
0xc6ddc N 0xc6de0
0xee7c0 T 0xaf52c
0xee7c0 T 0xaf52c
0xb1663c N 0xb16640
0xee7c0 T 0xaf52c
0xb1663c T 0xe7f2b0f8
0xc6ddc N 0xc6de0
0xb1663c N 0xb16640
0xb1663c T 0xe7f2b0f8
0xb1663c T 0xe7f2b0f8
0xc6ddc N 0xc6de0
0xee7c0 T 0xaf52c
0xee7c0 T 0xaf52c
0xb1663c N 0xb16640
0xb1663c T 0xe7f2b0f8
0xc6ddc N 0xc6de0
0xee7c0 T 0xaf52c
flush_num: 27, br_num: 47, size: 936b
